content/en/docs/reference/ports-and-protocols.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Ports and Protocols
+content_type: reference
+weight: 50
+---
+
+When running Kubernetes in an environment with strict network boundaries, such 
+as on-premises datacenter with physical network firewalls or Virtual 
+Networks in Public Cloud, it is useful to be aware of the ports and protocols 
+used by Kubernetes components
+
+## Control plane
+
+| Protocol | Direction | Port Range | Purpose                 | Used By                   |
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Inbound   | 6443*      | Kubernetes API server   | All                       |
+| TCP      | Inbound   | 2379-2380  | etcd server client API  | kube-apiserver, etcd      |
+| TCP      | Inbound   | 10250      | Kubelet API             | Self, Control plane       |
```

Comment:
The control plane only listens on 10250 if it's running on a node. There are Kubernetes clusters that pass all the tests where the control plane hosts aren't listening on this port.

Reply:
We could add a caveat about this after “Although etcd ports are included in control plane section, you can also host your own etcd cluster externally or on custom ports.”

Reply:
Does this come under general best practice, of "ports that are not used should not be open" ? Wondering if adding more caveats like this could confuse the reader.

Reply:
:shrug: I don't think we have even weak guidelines on that. Happy for you to decide here @PushkarJ !